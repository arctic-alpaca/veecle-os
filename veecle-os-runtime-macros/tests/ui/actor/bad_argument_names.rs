#[derive(Debug, PartialEq, Clone, Default, veecle_os_runtime::Storable)]
pub struct Sensor(pub u8);

#[veecle_os_runtime_macros::actor]
async fn macro_test_actor(
    Reader { .. }: veecle_os_runtime::Reader<'_, Sensor>,
) -> std::convert::Infallible {
    unreachable!("We only care about the code compiling.");
}

fn main() {}
