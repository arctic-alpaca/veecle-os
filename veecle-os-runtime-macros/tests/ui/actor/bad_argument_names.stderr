error: arguments must have a simple identifier pattern (e.g. `reader: Reader<..>`)
 --> tests/ui/actor/bad_argument_names.rs:6:5
  |
6 |     Reader { .. }: veecle_os_runtime::Reader<'_, Sensor>,
  |     ^^^^^^
